frameworks/kafka/docs/upgrade.md

```diff
@@ -0,0 +1,92 @@
+---
+post_title: Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+menu_order: 10
+feature_maturity: preview
+enterprise: 'no'
+---
+
+# Upgrade Kafka from 0.10.2.1 to 0.11.0.0
+
+This document explains upgrading the DC/OS Kafka service from 0.10.2.x to 0.11.0.0 version. Please follow the Kafka [upgrade](https://kafka.apache.org/documentation/#upgrade) documentation for more information. 
+
+The rolling upgrade procedure (from 0.10.2.x to 0.11.0.0) is explained below. This is the recommended way of upgrading to 0.11.0.0 in order to guarantee no downtime while upgrading Kafka.
+
+
+1. Upgrade the code to the new version:
+   * Upgrade Kafka software to 0.11.0.0.
+   * Restart the brokers one at a time.    
+   * Wait until the entire cluster is upgraded.
+
+2. Change inter.broker.protocol.version to 0.11.0.0:
+
+   * Edit server.properties files and change inter.broker.protocol.version to 0.11.0.0.
+   * Do not change log.message.format.version
+   * Restart the brokers one by one for the new protocol version to take affect
+
+3. Change log.message.format.version to 0.11.0:
+
+   * Edit server.properties files and change log.message.format.version to 0.11.0.
+   * Restart the brokers one by one for the new log format version to take affect
+
+
+
+The new DC/OS Kafka package that we will be upgrading to has the following default settings:
```

Comment:
Let's be explicit about package versions everywhere, please, to avoid confusion.